node_js/node_js.yml

```diff
@@ -0,0 +1,381 @@
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: burfect-shannon-53e001
+spec:
+    color: '#34bb55'
+    name: Node.js
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Variable
+metadata:
+    name: naughty-wiles-93e003
+spec:
+    language: flux
+    name: Node_Service
+    query: "import \"influxdata/influxdb/v1\"\r\nv1.tagValues(bucket: \"iot_center\",
```

Comment:
Your dashboard queries hard code "iot_center" for the bucket, so your template should include a bucket with that name.

This is the same bucket name as the IoT Center template, should it be using a unique bucket name for the Node.js template?